src/data/gen2Banner-urls.ts

```diff
@@ -29,57 +57,196 @@ export const GEN2BANNER_URLS = [
   '/javascript/build-a-backend/graphqlapi/troubleshooting/',
+  '/react/build-a-backend/graphqlapi/troubleshooting/',
+  '/angular/build-a-backend/graphqlapi/troubleshooting/',
+  '/vue/build-a-backend/graphqlapi/troubleshooting/',
+  '/nextjs/build-a-backend/graphqlapi/troubleshooting/',
   '/javascript/tools/cli/graphqlapi/directives-reference/',
+  '/angular/tools/cli/graphqlapi/directives-reference/',
+  '/react/tools/cli/graphqlapi/directives-reference/',
+  '/nextjs/tools/cli/graphqlapi/directives-reference/',
+  '/vue/tools/cli/graphqlapi/directives-reference/',
   '/javascript/build-a-backend/auth/',
+  '/angular/build-a-backend/auth/',
+  '/react/build-a-backend/auth/',
+  '/nextjs/build-a-backend/auth/',
+  '/vue/build-a-backend/auth/',
   '/javascript/build-a-backend/auth/set-up-auth/',
+  '/react/build-a-backend/auth/set-up-auth/',
+  '/angular/build-a-backend/auth/set-up-auth/',
+  '/vue/build-a-backend/auth/set-up-auth/',
+  '/nextjs/build-a-backend/auth/set-up-auth/',
   '/javascript/build-a-backend/auth/enable-sign-up/',
+  '/react/build-a-backend/auth/enable-sign-up/',
+  '/angular/build-a-backend/auth/enable-sign-up/',
+  '/vue/build-a-backend/auth/enable-sign-up/',
+  '/nextjs/build-a-backend/auth/enable-sign-up/',
   '/javascript/build-a-backend/auth/manage-user-session/',
+  '/react/build-a-backend/auth/manage-user-session/',
+  '/angular/build-a-backend/auth/manage-user-session/',
+  '/vue/build-a-backend/auth/manage-user-session/',
+  '/nextjs/build-a-backend/auth/manage-user-session/',
   '/javascript/build-a-backend/auth/add-social-provider/',
+  '/react/build-a-backend/auth/add-social-provider/',
+  '/angular/build-a-backend/auth/add-social-provider/',
+  '/vue/build-a-backend/auth/add-social-provider/',
+  '/nextjs/build-a-backend/auth/add-social-provider/',
   '/javascript/build-a-backend/auth/manage-user-profile/',
+  '/react/build-a-backend/auth/manage-user-profile/',
+  '/angular/build-a-backend/auth/manage-user-profile/',
+  '/vue/build-a-backend/auth/manage-user-profile/',
+  '/nextjs/build-a-backend/auth/manage-user-profile/',
   '/javascript/build-a-backend/auth/password-management/',
+  '/react/build-a-backend/auth/password-management/',
+  '/angular/build-a-backend/auth/password-management/',
+  '/vue/build-a-backend/auth/password-management/',
+  '/nextjs/build-a-backend/auth/password-management/',
   '/javascript/build-a-backend/auth/manage-mfa/',
+  '/react/build-a-backend/auth/manage-mfa/',
+  '/angular/build-a-backend/auth/manage-mfa/',
+  '/vue/build-a-backend/auth/manage-mfa/',
+  '/nextjs/build-a-backend/auth/manage-mfa/',
   '/javascript/build-a-backend/auth/delete-user-account/',
+  '/react/build-a-backend/auth/delete-user-account/',
+  '/angular/build-a-backend/auth/delete-user-account/',
+  '/vue/build-a-backend/auth/delete-user-account/',
+  '/nextjs/build-a-backend/auth/delete-user-account/',
   '/javascript/build-a-backend/auth/server-side-rendering/',
```

Comment:
This was moved - including JS - this should be added to:
[platform]/build-a-backend/server-side-rendering/nextjs/